src/angular/table/table.md

```diff
@@ -477,6 +477,36 @@ The flag affects the sequence at runtime, not at definition time.
 </ng-container>
 ```
 
+### Table Cell Actions
+
+It is possible to define up to three action buttons (or links) for one row.
```

Comment:
Is there a specific reason/limitation as to why we only support up to three?

Reply:
It comes from design specification. But we don‘t need to mention it here. What do you think?